src/app/client/src/app/modules/dashboard/components/report/report.component.ts

```diff
@@ -25,7 +26,7 @@ export class ReportComponent implements OnInit {
 
   public report: any;
   public showSummaryModal = false;
-  public report$;
+  public report$: any;
```

Comment:
Please try to avoid "any" type as much as possible and use interface

Reply:
This is applicable to all the components wherever you've used any as the type

Reply:
resolved